nicesite.so.hosting.json

```diff
@@ -0,0 +1,24 @@
+{
+  "providerId": "nicesite.so",
+  "providerName": "nicesite",
+  "serviceId": "website",
```

Comment:
Automatic check failed because of file name mismatch to serviceId:
`AssertionError: File name nicesite.so.hosting.json not matching the pattern. Expected nicesite.so.website.json`
Please either rename the file or adjust serviceId accordingly

See README.md:
> Template Naming Format
> Templates should be named according the following pattern: providerId.serviceId.json